pkg/virtctl/vnc/vnc.go

```diff
@@ -65,6 +65,10 @@ func (o *VNC) Run(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	if value, ok := os.LookupEnv("KUBECTL_PLUGINS_CURRENT_NAMESPACE"); ok {
```

Comment:
Hm the pattern we have in the commands is very error prone.

I guess the best thing would be to always rely on the pflag namespace value and set it in `RunE` like described for the other flags.